src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,41 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
Consider whether the `StorageDao` instance should be initialized in a `@BeforeAll` setup if it's not going to change between tests, or `@BeforeEach` if you need a fresh state before each test. This can improve the design of your tests and clarify the lifecycle of your test objects.